conductor-config-ci.yaml

```diff
@@ -1,3 +1,6 @@
+dpki:
+  device_seed_lair_tag: ci
+  no_dpki: true
```

Comment:
I think we should make a decision about whether we're testing with this on or off. Maybe off makes sense right now because Holo are configuring their system that way for now. 

But equally, you could argue that this is the future direction and we should have at least one test that turns it on before Holo start using it.

Thoughts from everyone else?

Reply:
I guess the simplest thing to do would be to not commit this change for now. Anyone can modify or use separate configs locally for whatever test scenario they want to use, and we can figure out what the default should be separately.

Reply:
done